scripts/docker_cmake

```diff
@@ -6,7 +6,7 @@ ARGS=(
     -D CMAKE_BUILD_TYPE=Debug
     -D BUILD_SHARED_LIBS=ON
     ### COMPILERS AND FLAGS ###
-    -D CMAKE_CXX_FLAGS="-Wall -Wpedantic ${CUDA_CXX_FLAGS}"
+#    -D CMAKE_CXX_FLAGS="-Wall -Wpedantic"
```

Comment:
We pass these flags with `DataTransferKit_CXX_FLAGS` and we already get enough warnings from other packages from Trilinos via the headers.  What do you think?  Should we uncomment?